src/services/userService.js

```diff
@@ -1,13 +1,12 @@
+import { user } from './mocks/userData';
+
 export function logIn({login, password}) {
```

Comment:
You can use here default values as ({ login = '', password = '' })

Reply:
done